### ISSUE TITLE
Failure download git-annex's files from UI on master branch

### ISSUE THREAD

**Issue by ivis-tsukioka** [G-Node/gogs#125]
I had built GIN from recent master branch on my local machine to reseach GIN.
But I  could not downloads git-annex's files(PDF) on UI (push download botton).
Next, building GIN  from recent "live" branch on local, I can downloads git-annex's files on UI.

Comparing master and live branch, 
I understood that 
1. "master" use gogs/git-module, on the other hand, "live" use G-Node/git-module.
2.  It is difference on internal/route/repo/download.go between two branch.
3.  most recent release tag is derived from live branch.

I have some questions.
1. Why do master branch use  gogs/git-module?
2. Do you plan where branch is next release ? 
3. In future, will master branch be able to download git-annex's files on UI ?

**Comment by mpsonntag** [comment]
Hi Ivis,

the master branch is our current "hands on" branch and not necessarily available for a full build. Please always use the latest tagged version on the "live" branch. Currently this would be the [gin-live-2020-10-24 tag](https://github.com/G-Node/gogs/tree/gin-live-2020-10-24).

**Comment by ivis-tsukioka** [comment]
Thank you for your answer.
I understood that we should use "live" branch for full build now.

In G-Node/gogs, master branch have be modificted during  25/Nov./2020 to 01/Sep./2021, but live branch had be modificated   
until 23/Nov.2020.

We have a plan that is development of  modified GIN, using our repository(GIN fork) is forked from G-Node/gogs.
And in future, We think that your modification on G-Node/gogs marge into our repository(GIN fork).
We are hesitating that I should select branch(master or live) to fork  to our repository.
Is it right that we should fork your live branch  to our repository?

**Comment by mpsonntag** [comment]
Yes, to ensure that you are starting from a working server state, it would be best to work with the live branch.

**Comment by ivis-tsukioka** [comment]
In G-node/gogs, is it master branch  as development branch, and live branch as release branch ?
And In future, is it right that the modificated code on master branch merge into live branch ?

**Comment by mpsonntag** [comment]
For the immediate future the branch setup will remain this way, yes. But the best option would be to always use the latest git `tag` ("gin-live-YYYY-MM-DD") that corresponds to the latest working release. This way, even if the branches might change, you will always end up with a working release.
